homu/server.py

```diff
@@ -123,7 +123,8 @@ def callback():
         })
     except Exception as ex:
         logger.warn('/callback encountered an error during github oauth callback')
-        lazy_debug(logger, lambda: 'github oauth callback err: {}'.format(ex))
+        s = 'github oauth callback err: {}'.format(ex)
```

Comment:
This makes the thing un-lazy, since `s` is unconditionally evaluated. Set s to `lambda: ...` and use it directly